### ISSUE TITLE
Enable Renovate app

### ISSUE THREAD

**Issue by jan-molak** [protractor-cucumber-framework/protractor-cucumber-framework#202]
Hi @flaviencathala!

Would you consider enabling [Renovate app](https://github.com/apps/renovate) on this repo to keep Serenity/JS deps up to date? It's free for open-source software and I've added the [required config](https://github.com/protractor-cucumber-framework/protractor-cucumber-framework/blob/master/.github/workflows/renovate.json) in my last PR, but I think we still need to enable the bot itself.

Give me a shout if I can be of any help with that!

Best,
Jan

**Comment by flaviencathala** [comment]
Hey @jan-molak, I will ask for Admin access on this repo to enable Renovate and automated releases.

**Comment by jan-molak** [comment]
@flaviencathala - I've just activated Renovate, so we should see some PRs coming in shortly;

Do you think we should configure [`semantic-release`](https://github.com/semantic-release/semantic-release) for  `protractor-cucumber-framework`? Renovate app detects conventional commits, so we could use that to do automated releases.

**Comment by flaviencathala** [comment]
@jan-molak yes good idea, it would fully automate the release after a PR merge.

**Comment by jan-molak** [comment]
Cool! Would you mind running the [setup](https://github.com/semantic-release/semantic-release/blob/master/docs/usage/getting-started.md)? It needs someone with access to the npm registry. I'm happy to be added to the project on npmjs.com too, if you'd like?

**Comment by flaviencathala** [comment]
@jan-molak I may find the time to do it this weekend. I gave you the access to the npm package this morning - at least I hope it was you.

**Comment by jan-molak** [comment]
Haha, oh my, I must have missed the notification. Yeah, I have it, thanks!
